vocode/streaming/transcriber/google_transcriber.py

```diff
@@ -16,6 +16,7 @@
 from vocode.streaming.utils import create_loop_in_thread
 
 
+# TODO: make this nonblocking so it can run in the main thread, see speech.TextToSpeechAsyncClient
```

Comment:
Actually the client for transcription is 
`from google.cloud.speech_v1.services.speech.async_client import SpeechAsyncClient`

`TextToSpeechAsyncClient` is for the synthesizer. Apologies for the confusion.